src/class-elements/syntax/valid/cls-decl-elements-valid-syntax-export.template

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-ClassElement
+name: class declaration module export
+path: language/statements/class/elements/syntax/valid/export-
+flags: [module]
+features: [class]
+---*/
+
+export class C /*{ heritage }*/{
+  /*{ elements }*/
+}
```

Comment:
I still believe a single test containing a `export ClassDeclaration` is sufficient. This does not observe special syntax for elements in this position.